doc/api/errors.md

```diff
@@ -575,6 +586,7 @@ found [here][online].
 [domains]: domain.html
 [event emitter-based]: events.html#events_class_eventemitter
 [file descriptors]: https://en.wikipedia.org/wiki/File_descriptor
+[Node.js Error Codes]: #nodejs-error-codes
```

Comment:
@jasnell Yes, I realized it after a while. Shouldn't we let the tools take care of deciding the ids?